### DISCUSSION TITLE
Temporal skill assesment

### DISCUSSION THREAD

**Discussion by ecomodeller** [DHI/modelskill#11] [Ideas]
Add functionality to create an aggregated time series (across locations) to be able to assess temporal variation in model skill.

Consider temporal smoothing.

In the example below, where four different model runs are compared, the best model varies over time.
![image](https://user-images.githubusercontent.com/614215/107617890-f6478d80-6c50-11eb-9b7f-97a6dfabad2c.png)

**Comment by jsmariegaard** [comment]
See #33
